### DISCUSSION TITLE
Share your favorite moment/memory from the past 7 years with Kata Containers!

### DISCUSSION THREAD

**Discussion by ildikov** [kata-containers/kata-containers#10623] [General]
The Kata Containers project was announced 7 years ago, on December 5th, 2017: https://www.openstack.org/news/view/365/kata-containers-project-launches-to-build-secure-container-infrastructure

It is a great opportunity to reflect on what inspired you to join the project and how your experience have been within the community!

What is your favorite Kata Containers story? Do you have a special moment or memory about the project that you can share? What did inspire you to look into the project at first, and what made you stay with the community?

**Comment by stevenhorsman** [comment]
Thanks to @Apokleos & @ildikov was this idea, it's very cool.

I started with the project over 3 years ago and was just trying out kata, wanting to get a dev, build and test environment set-up and hit some issues with some out-of-date docs 😱! I decided to push some fixes and made a lot of mistakes ignoring the patch format and then screwing up my commits and I was lucky enough that @fidencio took the time and interest to help me out in a really patient and friendly way. That pretty much sets the pattern for my best moments - working with the great people in the community. Whether it was working main late nights with @fidencio and @wainersm on the initial confidential containers kata releases, or discussions in slack, AC & CI meetings - the technology is good, but it's the people that make the project what it is. So a big thank you to everyone who has welcomed me into the community and taught me so much along the way!

**Comment by gnawux** [comment]
![DSC04641](https://github.com/user-attachments/assets/f4da2765-4bda-45f2-8904-c3a48d3a0a77)
The day we launched Kata Containers :)

**Comment by fidencio** [comment]
I've spent a reasonable amount of my time around and surrounded by this community in the past 4 years (more than half the time of the project, and a good amount of those years in an architecture committee seat).

As @stevenhorsman mentioned about being well received and this setting the pattern for the best moments, I can only say that I've been trying to receive folks as well as I was received when I firstly joined the IRC channel ;-)).

I remember very early discussions (not always with agreements ;-)) with @bergwolf, @gnawux, @egernst, @devimc, and a few other folks who've moved away to different projects; and also very late nights working on CoCo stuff with "newer members" like @stevenhorsman himself and others mentioned (who, nowadays, almost feel like they're veterans already ;-)).

I've made friends here, I've probably made some enemies here as well, but yeah, cannot say anything but ... this has been a very interesting learning experience, and a very nice (and somehow selfish) opportunity to be surrounded by folks who're better than me on what I do! :-)

Thank y'all, Kata Containers community!
